tests/fast/stages/s2_networking_a_peering/tftest.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module: fast/stages/2-networking-a-peering
+common_tfvars:
+  - common.tfvars
+
+tests:
+  stage:
+    extra_files:
+      - ../../plugins/2-networking-serverless-connector/*.tf
```

Comment:
if you want to test the plugin, you have to set `var.serverless_connector_config` in the tfvars

Reply:
not really, the default kicks in

Reply:
![image](https://user-images.githubusercontent.com/66007/227505293-b2a2a38c-ba83-4cda-babb-40fec55cddb7.png)